RealmSwift/List.swift

```diff
@@ -361,6 +361,14 @@ extension List: MutableCollection {
     }
 }
 
+// MARK: - Hashable
+
+extension List: Hashable {
```

Comment:
This is not a valid definition of Hashable for List, and such a thing is probably not possible. Using a List as a dictionary key does not really make any sense too.

Reply:
Same as Map